assets/js/app.js

```diff
@@ -496,6 +499,9 @@ global.MonsieurBizRichEditorManager = class {
   write() {
     this.input.value = (this.uiElements.length > 0) ? JSON.stringify(this.uiElements) : '';
     this.drawUiElements();
+    document.dispatchEvent(new CustomEvent('monsieurBizRichEditorWriteComplete', {
```

Comment:
```suggestion
    document.dispatchEvent(new CustomEvent('mbiz:rich-editor:write-complete', {
```
Because [convention seems more like this](https://gomakethings.com/custom-event-naming-conventions-in-vanilla-js)